### DISCUSSION TITLE
Add optional baseline assumption that future Rt will regress to one

### DISCUSSION THREAD

**Discussion by nikosbosse** [epiforecasts/EpiNow2#280] [Ideas] [answered]
Currently EpiNow2 makes predictions by assuming Rt will be constant in time. One could argue that based on what we observed in the past it would make more sense to assume that the pandemic is a control system where Rt will slowly revert back to one in the future. Maybe an option for that would be nice.

**Comment by seabbs** [comment]
So this seems like a valid user choice and one that is currently supported using the simulation tooling. As part of the `epinow` pipeline this has been integrated using `forecast_infections` so the first stop is to check if that would allow you to do what. you want and if not explore what else you would need.

**Comment by sbfnk** [comment, marked as answer]
One way to do this would be to treat EpiNow2 as a tool that doesn't provide a forecasting model (which it doesn't really) -- what it does is offer a few ways to extend the data forward.

An external package to this could implement a range of forecast models (involving e.g. susceptible depletion, vaccination, regression to 1 etc.) that feeds into the existing framework for external forcing of R (in `forecast_infections`, I presume).

**Reply by seabbs** [reply]
:+1: This seems like the time to wake EpiSoon back up 😆

**Comment by seabbs** [comment]
Looking at the code for `forecast_infections` (of which I have no memory). This seems doable as it stands by passing in a very custom `forecast_model` but the simulation tools that in there are from `EpiSoon` vs the better (😆) ones available in `simulate_infections`. At some point `forecast_infections` needs a rewrite to use `simualate_infections` especially if this is an approach someone is taking (I have not seen anyone doing this).
